public/app/features/alerting/unified/components/folder-bulk-actions/MenuItemPauseFolder.tsx

```diff
@@ -23,25 +23,26 @@ export function FolderActionMenuItem({ folderUID, onActionSucceed, executeAction
   const notifyApp = useAppNotification();
   const label =
     action === 'pause'
-      ? t('alerting.folder-bulk-actions.pause.button.label', 'Pause evaluation')
-      : t('alerting.folder-bulk-actions.unpause.button.label', 'Unpause evaluation');
+      ? t('alerting.folder-bulk-actions.pause.button.label', 'Pause all rule evaluation')
+      : t('alerting.folder-bulk-actions.unpause.button.label', 'Resume all rule evaluation');
```

Comment:
We call it "resume" on individual rule level, so makes sense to be consistent here. WDYT?

Reply:
yes